roles/setup/tasks/setup-CentOS.yml

```diff
@@ -29,10 +30,3 @@
     - qemu-img
     - tar
 
-- name: Install cloud-localds from Canonical's cloud-utils
```

Comment:
Why did you drop this? You need this on RHEL/CentOS because there is no cloud-localds otherwise.